### DISCUSSION TITLE
music/hermod/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/music#2] [General]
# music/hermod/

https://shanenull.com/music/hermod/

**Comment by shane0** [comment]
https://squarp.net/legacy/hermod/manual/

**Comment by shane0** [comment]
hapax DEFAULT is the name of the default project

hermod is _

**Comment by shane0** [comment]
# hermod+

- do I need a hermod+ 
- of course I do but first I need to sell mine

**Comment by shane0** [comment]
# lfo out

The Squarp Hermod (and its successor, Hermod+) is a powerful Eurorack module that excels at sequencing, but also has excellent LFO capabilities due to its flexible "effect" system. Here's how you can generate and output LFOs from your Hermod:

**Hermod's LFO Capabilities:**

* **LFO as an Effect:** Unlike some modules with dedicated LFO sections, Hermod implements LFOs as one of its real-time effects that can be applied to any of its 8 tracks (16 tracks on Hermod+). Each track can have up to 8 effects stacked, meaning you can have multiple LFOs (or LFOs chained with other effects) per track.
* **Dedicated LFO Tracks:** You can dedicate a track solely to generating an LFO. This is a common and effective way to use Hermod as a multi-channel LFO generator for your Eurorack system.
* **CV Output:** Hermod has 8 CV outputs (and 8 Gate outputs). You can route the LFO effect on a track to one of these CV outputs to send the LFO signal to other modules in your rack. The CV outputs range from -5V to +5V (or 0V to +5V depending on configuration/effect parameters), which is standard for Eurorack modulation.
* **MIDI Output:** Hermod can also send LFOs as MIDI CC (Control Change) messages, Pitch Bend, or Aftertouch via its MIDI outputs (TRS MIDI, USB Host, USB Device). This is useful for modulating external MIDI hardware or software instruments.

**How to Set Up an LFO on Hermod (General Steps):**

1.  **Select a Track:** Go to the "Track" mode on your Hermod. Choose an empty track, or a track you wish to dedicate to LFO generation.
2.  **Configure Track Layout (for CV Output):**
    * Press 'X' (or click the encoder and select "layout") to enter the track configurator.
    * Set the track to "1 VOICE" and "MODULATION." This designates the track for sending out CV modulation signals.
3.  **Add the LFO Effect:**
    * Go to the "Effects" mode.
    * Select the track you're working on.
    * Add the "LFO" effect to the track's effect slot.
4.  **Configure LFO Parameters:**
    * **WAVE:** Choose your desired waveform (Sine, Triangle, Saw Up, Saw Down, Square, Random, etc.).
    * **RATE:** Set the speed of the LFO. This can be free-running or synchronized to Hermod's internal clock or an external clock.
    * **SYNC:** If desired, set the LFO to sync to the master clock at a specific division (e.g., 1/4, 1/8, 1/16).
    * **DEPTH/AMOUNT:** Adjust the amplitude of the LFO.
    * **PHASE (for multiple LFOs):** If you're using multiple LFOs, you can offset their phases to create more complex modulation.
    * **DESTINATION (for MIDI Output):** If you want the LFO to control a MIDI parameter, you can set the destination to Pitch Bend, Modulation (CC1), or Aftertouch. For arbitrary MIDI CCs, you might need to use the ModMatrix in conjunction with a MIDI Output effect (see below).

5.  **Route the LFO to an Output:**
    * For **CV Output**: The LFO will automatically be sent out of the CV output associated with the track you've configured as "MODULATION." Each of Hermod's 8 CV/Gate outputs corresponds to a track. So, if you set Track 1 to "MODULATION," its LFO will appear at CV Out 1.
    * For **MIDI Output**: If you set the LFO's destination to a MIDI message (like CC1), ensure your track's MIDI output is correctly configured to the desired MIDI port (TRS A, TRS B, USB Host, USB Device).

**Important Tips and Considerations:**

* **"Stepping" and Glide Effect:** Some users have reported "stepping" in CV output when using LFOs, especially for smoother waveforms like sine waves. To counteract this, you can often add a **"Glide" effect** *after* the LFO effect in the effect chain on that track. This will smooth out the transitions.
* **ModMatrix for Advanced Routing:** Hermod's ModMatrix is incredibly powerful. You can use it to:
    * **Control LFO parameters:** Use a CV input or another MIDI CC to modulate the LFO's rate, depth, or even waveform.
    * **Map LFO to other destinations:** If the direct LFO destination options aren't enough, you can set an LFO to control a MIDI message (like CC1), and then use the ModMatrix to map that CC1 to another effect parameter on the *same or different track*, or to a different arbitrary MIDI CC via a MIDI output effect.
* **Poly-Morphic Layouts:** Hermod's "polymorphic" track layout allows you to use your 8 (or 16) outputs flexibly. You can have a mix of CV/Gate notes, modulation, velocity, and aftertouch outputs across your tracks, adapting Hermod to your specific patching needs.
* **Chaining LFOs:** You can stack multiple LFO effects on a single track to create more complex LFO shapes by combining their outputs.
* **Save Projects/Templates:** Once you have a useful LFO setup, save it as part of a project or create templates for quick recall.

By leveraging Hermod's effect system and flexible routing, you can turn it into a versatile multi-LFO module for your modular system.

**Comment by shane0** [comment]
# troubleshooting

**Reply by shane0** [reply]
# track 1 not playing unless active

It sounds like your Squarp Hermod (or Hermod+) track 1 is set to a mode where it only outputs live input or generative effects, and *not* its internal sequence, unless it's the actively selected track. This is a common and usually intentional feature, but it can be confusing if you're not expecting it!

The most likely culprit is the **"MUTE AFFECTS INPUT"** setting, or simply how "Mute" works on the Hermod.

Here's how to troubleshoot and fix this:

**1. Check the Track Mute Status:**

* **In TRACK mode:** Hermod has a dedicated "TRACK" mode (often one of the main mode buttons). In this mode, the 8 track pads represent the mute status of each track.
* **Muted tracks:** A muted track's pad will typically be **dimmed** or **flashing/greyed out on the screen** (visuals may vary slightly between Hermod and Hermod+ or firmware versions).
* **Unmute:** Simply **press the track's pad** to toggle its mute status. If it's muted, pressing it will unmute it, and it should then play its sequence whether it's highlighted or not.

**Why Muting Can Be Confusing:**

Squarp added a "MUTE AFFECTS INPUT" setting in later firmware updates (especially for Hermod+).

* **"MUTE AFFECTS INPUT: YES" (Legacy behavior):** When a track is muted, *all* its output is silenced, including sequenced notes, live MIDI/CV input, and effects. This is the older, more common mute behavior. If your track is muted and set like this, it won't play anything.
* **"MUTE AFFECTS INPUT: NO" (Default on Hermod+):** When a track is muted, *only the sequenced data* is muted. Live inputs (from a keyboard connected to MIDI IN, or CV IN) and generative effects (like an arpeggiator or randomizer) *will still pass through* and be sent to the output. This is very handy for live performance, allowing you to mute a sequence but still jam over it or have an arpeggiator continue.

    * **This is likely what you're experiencing:** If your track 1 is muted with "MUTE AFFECTS INPUT: NO", its *sequence* won't play, but if you select it (highlight it), and you're sending live MIDI to it, *that* live MIDI will sound. Or if you have a generative effect on the track, that might still sound.

**How to Check/Change "MUTE AFFECTS INPUT":**

1.  **Go to TRACK mode.**
2.  **Hold down the track pad for track 1** (the one that's giving you trouble) to open its specific track settings.
3.  **Scroll through the parameters** until you find "MUTE AFFECTS INPUT".
4.  **Change it to "YES"** if you want the track to be completely silent when muted. Or, if you want it to play its sequence, simply **unmute the track** in TRACK mode.

**Other Potential (Less Likely) Causes:**

* **Track Pattern Empty:** While you say it plays when highlighted, it's worth double-checking if there's actually a pattern recorded on the track for the current sequence. Go to STEP mode for that track and see if there are any notes.
* **Sequence Muting:** It's possible to mute tracks on a *sequence-by-sequence* basis. If you're in SEQ mode and have multiple sequences, a track might be muted in one sequence but unmuted in another.
    * In **SEQ mode**, if you press and hold a sequence pad, you'll often see options related to track mute states for that specific sequence.
* **Effects Muting:** An effect on the track could be muting the output of the sequence. Go to EFFECTS mode, select track 1, and ensure none of the effects in its chain are muted.
* **CV/MIDI Output Configuration:** Double-check that the track's CV/MIDI output is correctly assigned in the track settings (hold the track pad, then scroll to OUTPUT PORT and OUTPUT CHANNEL/DESTINATION). If it's set to "OFF" or an incorrect port/channel, it won't be heard.

**Summary of Action Plan:**

1.  **Go to TRACK mode.**
2.  **Look at the pad for Track 1.** Is it dimmed or flashing? If so, press it to unmute.
3.  If unmuting doesn't work, **hold down the Track 1 pad** to enter its settings.
4.  **Find "MUTE AFFECTS INPUT"** and understand its setting. If it's "NO", that explains why live input still plays when muted.
5.  Confirm the **OUTPUT PORT** and **OUTPUT CHANNEL/DESTINATION** settings are correct for your receiving module.

Hopefully, one of these steps will resolve your issue and get your Hermod track playing as expected!
